ReactNativeClient/lib/services/SearchEngine.js

```diff
@@ -7,6 +7,7 @@ const ItemChangeUtils = require('lib/services/ItemChangeUtils');
 const { pregQuote, scriptType } = require('lib/string-utils.js');
 const removeDiacritics = require('diacritics').remove;
 const { sprintf } = require('sprintf-js');
+const { map, pickBy, identity, isEmpty } = require('lodash');
```

Comment:
Please don't use lodash as we prefer not to depend on it for now.

Reply:
Done. They are removed now.